src/components/Loading.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navbar } from "./Navbar";
+
+import styles from "./Layout.module.css";
+
+export function Loading({ activePage = '' }: any) {
```

Comment:
This isn't a problem, but maybe we should have loading take the active page as a parameter? Although it might make sense to allow for no active page, I don't know if that's ever a case for us.